### DISCUSSION TITLE
Selecting multiple correspondents

### DISCUSSION THREAD

**Discussion by geomule** [jonaswinkler/paperless-ng#833] [General]
Hi folks,

really enjoying Paperless. Already uploaded 1500 documents in leas than 2 weeks. I'm trying to understand whether I can save a view that allows me to look at all my bills from the last 3 months. Seems like you can select multiple tags but not multiple correspondents? I guess the way around this is to optimise the use of tage but just checking there is no way that I can select multiple correspondents?

thanks

**Comment by shamoon** [comment]
Yea you’re correct it’s not supported at the moment. PR https://github.com/jonaswinkler/paperless-ng/pull/399 is in limbo but might help if/when it’s merged.

But also do you have a doc type for “Bills”? That’s how I’d do this in my setup (or a tag too of course)
